### PR TITLE
Feat/965 track emphasized text msword

### PR DESCRIPTION
**Pull request by christinestraub**
### Summary
Update `partition_doc` and `partition_docx` to track emphasized text in metadata when text is bold or emphasized (bold/italic formatting) in MS Word.
- add functionality to track emphasized text (`bold/italic` formatting) from a paragraph
- add functionality to track emphasized text (`bold/italic` formatting) from a table

### Testing
```
filename = "example-docs/fake-doc-emphasized-text.docx"
elements = partition_docx(filename=filename)

filename = "example-docs/fake-doc-emphasized-text.doc"
elements = partition_doc(filename=filename)
```
---
Addresses [#965](https://github.com/Unstructured-IO/unstructured/issues/965) for MS Word

### CODE REVIEW

File: test_unstructured_ingest/expected-structured-output/onedrive/utic-test-ingest-fixtures/tests-example.json
Comment (MthwRobinson) [inline test_unstructured_ingest/expected-structured-output/onedrive/utic-test-ingest-fixtures/tests-example.json:26]: Not sure why this changed but seems like an improvement 🤷 Might have been from #1009 (although ingest tests were passing before that merged)

**Review by MthwRobinson — APPROVED** [review]
Nice addition - thanks! LGTM once CI is passing.

File: test_unstructured_ingest/expected-structured-output/onedrive/utic-test-ingest-fixtures/tests-example.json
Comment (christinestraub) [reply test_unstructured_ingest/expected-structured-output/onedrive/utic-test-ingest-fixtures/tests-example.json:26]: > Not sure why this changed but seems like an improvement 🤷 Might have been from #1009 (although ingest tests were passing before that merged)

Yeah, that seems to have changed since #1009.

**Pull request merged**
